blueprints/factories/project-factory/main.tf

```diff
@@ -131,19 +144,18 @@ locals {
     coalesce(var.labels, {}), coalesce(try(var.defaults.labels, {}), {})
   )
   services = distinct(concat(var.services, local._services))
-  vpc = coalesce(var.vpc, {
-    host_project = null, gke_setup = null, subnets_iam = null
-  })
   vpc_cloudservices = (
-    local.vpc_gke_service_agent ||
+    var.vpc.gke_setup.enable_host_service_agent ||
     contains(var.services, "compute.googleapis.com")
```

Comment:
Anybody can give a reference to docs why do we grant `roles/compute.networkUser` to cloudservices if using Compute Engine? 

Should this be added to `sharedvpc-agent-iam.yaml`?

Reply:
MIGs managing autoscaling VMs